src/Url.php

```diff
@@ -657,7 +657,7 @@ public function hasIdn(): bool
     }
 
     /**
-     * Returns true if the current instance URL is equal to the URL you want to compare.
+     * Returns true if the current instance `Url` is equal to the URL you want to compare.
```

Comment:
I'd revert this, because it means more like "the URL represent by the current instance", so it doesn't really refer to the class/instance itself.

Reply:
```suggestion
     * Returns true if the current instance URL is equal to the URL you want to compare.
```